backend/docker.go

```diff
@@ -29,6 +29,7 @@ var (
 		"CERT_PATH":       "directory where ca.pem, cert.pem, and key.pem are located (default \"\")",
 		"CMD":             "command (CMD) to run when creating containers (default \"/sbin/init\")",
 		"MEMORY":          "memory to allocate to each container (0 disables allocation, default \"4G\")",
+		"SHM":             "/dev/shm to allocate to each container (0 disables allocation, default \"64MB\")",
```

Comment:
What I mean is this code:

``` Go
shm := uint64(1024 * 1024 * 64)
if cfg.IsSet("SHM") {
	if parsedShm, err := humanize.ParseBytes(cfg.Get("SHM")); err == nil {
		shm = parsedShm
	}
}
```

We're passing in a byte count to docker, not a string. Docker may parse MB as 1024-based, but `humanize.ParseBytes` does not, so I think either changing the doc string to "64 MiB" or changing the default value to `64 * 1000 * 1000` would be best (probably the former, since memory things are usually 1024-based).